src/runtimes/node/in_source_config/index.ts

```diff
@@ -61,6 +62,25 @@ export const findISCDeclarationsInPath = async (
   return findISCDeclarations(source, { functionName, featureFlags, logger })
 }
 
+/**
+ * Normalizes method names into arrays of uppercase strings.
+ * (e.g. "get" becomes ["GET"])
+ */
+const normalizeMethods = (method: unknown, name: string): string[] | undefined => {
+  const methods = Array.isArray(method) ? method : [method]
+
+  // eslint-disable-next-line @typescript-eslint/no-shadow
+  return methods.map((method) => {
+    if (typeof method !== 'string') {
+      throw new TypeError(
```

Comment:
Should this be a `FunctionBundlingUserError`, so that it's flagged as a user error?

Reply:
done in [015726f](https://github.com/netlify/zip-it-and-ship-it/pull/1539/commits/015726f6bd264238ec3c5555cbf03c0518a83f58)